### PR TITLE
Table in Connect added UPDATE command #907

### PR DESCRIPTION
**Pull request by amyshwang**
Adding a note indicating relevant version.
TBD hyperlink

### CODE REVIEW

File: docs/develop/connect.md [hunk 1]
```diff
@@ -3,17 +3,19 @@
...
-|Network Endpoint|Port|Inserting data|Querying data|
-|:---------------|:---|:-------------|:------------|
-|[Web Console](#web-console)|9000|SQL `INSERT`, CSV|SQL `SELECT`, charting|
-|[InfluxDB Line Protocol](#influxdb-line-protocol)|9009|High performance bulk insert|-|
-|[PostgreSQL wire protocol](#postgresql-wire-protocol)|8812|SQL `INSERT`|SQL `SELECT`|
-|[HTTP REST API](#http-rest-api)|9000|SQL `INSERT`, CSV|SQL `SELECT`, CSV|
+|Network Endpoint|Port|Inserting data|Querying data|Updating data|
+|:---------------|:---|:-------------|:------------|:-------------|
+|[Web Console](#web-console)|9000|SQL `INSERT`, CSV|SQL `SELECT`, charting| SQL `UPDATE`|
+|[InfluxDB Line Protocol](#influxdb-line-protocol)|9009|High performance bulk insert|-|SQL `UPDATE`|
```
Comment (amyshwang) [inline docs/develop/connect.md:9]: Should be "-"

File: docs/develop/connect.md [hunk 2]
```diff
@@ -3,17 +3,19 @@
 You can interact with a QuestDB database by connecting to one of its various
 network endpoints.
 
-|Network Endpoint|Port|Inserting data|Querying data|
-|:---------------|:---|:-------------|:------------|
-|[Web Console](#web-console)|9000|SQL `INSERT`, CSV|SQL `SELECT`, charting|
-|[InfluxDB Line Protocol](#influxdb-line-protocol)|9009|High performance bulk insert|-|
-|[PostgreSQL wire protocol](#postgresql-wire-protocol)|8812|SQL `INSERT`|SQL `SELECT`|
-|[HTTP REST API](#http-rest-api)|9000|SQL `INSERT`, CSV|SQL `SELECT`, CSV|
+|Network Endpoint|Port|Inserting data|Querying data|Updating data|
```
Comment (amyshwang) [inline docs/develop/connect.md:6]: Updating --> Modifying? To be aligned with https://questdb.io/docs/guides/modifying-data/

File: docs/develop/connect.md [hunk 3]
```diff
@@ -3,17 +3,19 @@
...
+|[Web Console](#web-console)|9000|SQL `INSERT`, CSV|SQL `SELECT`, charting| SQL `UPDATE`|
+|[InfluxDB Line Protocol](#influxdb-line-protocol)|9009|High performance bulk insert|-|SQL `UPDATE`|
+|[PostgreSQL wire protocol](#postgresql-wire-protocol)|8812|SQL `INSERT`|SQL `SELECT`|SQL `UPDATE`|
+|[HTTP REST API](#http-rest-api)|9000|SQL `INSERT`, CSV|SQL `SELECT`, CSV|SQL `UPDATE`|
 
 :::note
 
 All network ports may be [configured](/docs/reference/configuration).
 
+`UPDATE` is available from [QuestDB 6.4](blog/2022/05/31/questdb-release-6-4/).
```
Comment (amyshwang) [inline docs/develop/connect.md:17]: This link is not working.
Or linking to docs/guides/modifying-data/ instead, changing the linked text to [`UPDATE`]?

**Review by amyshwang — COMMENTED** [review]
Adding some comments before submitting another PR to confirm my understanding.

File: docs/develop/connect.md (hunk 3 above)
Comment (amyshwang) [reply docs/develop/connect.md:17]: Should be `/blog/2022/05/31/questdb-release-6-4/`

File: docs/develop/connect.md (hunk 2 above)
Comment (jerrinot) [reply docs/develop/connect.md:6]: hm, maybe I would merge Modifying and Insert into a single column. Something along these lines:
|Network Endpoint|Port|Inserting & Modifying data|Querying data|
|:---------------|:---|:-------------|:------------|
|[Web Console](#web-console)|9000|SQL `INSERT`, `UPDATE`, CSV|SQL `SELECT`, charting| SQL `UPDATE`|
|[InfluxDB Line Protocol](#influxdb-line-protocol)|9009|High performance bulk insert|-|
|[PostgreSQL wire protocol](#postgresql-wire-protocol)|8812|SQL `INSERT`, `UPDATE`|SQL `SELECT`|
|[HTTP REST API](#http-rest-api)|9000|SQL `INSERT`, `UPDATE`, CSV|SQL `SELECT`, CSV|

Othewise the table looks busy yet very repetitive:
image

If you keep your version then please remove `SQL UPDATE` from the Influx row. As Influx does not support UPDATE. Influx is for insertions only.

**Comment by jerrinot** [comment]
looks good to me. the Info table with 2 unrelated piece of information looks unusual, but I don't think it's a problem. I certainly do not know how to do it better. 

image

**Review by jerrinot — APPROVED** [review]

**Pull request merged**